app/models/child.rb

```diff
@@ -9,6 +9,7 @@ class Child < UuidApplicationRecord
 
   has_many :child_approvals, dependent: :destroy, inverse_of: :child, autosave: true
   has_many :approvals, through: :child_approvals
+  has_many :schedules, dependent: :destroy
```

Comment:
Seems straightforward, sets up one to many relationship child to schedules (child has many schedules).  Destroy means destroy schedules when child is destroyed I think?

Reply:
Yep!  Exactly that!